src/Data.hs

```diff
@@ -121,9 +122,10 @@ instance StringReferencingBinary CaptureData where
 -- 2 Using ListOfStringable
 -- 3 Add cDesktop
 -- 4 WindowData instead of 3-tuple
+-- 5 CompactNum
```

Comment:
Worth doing both with the same tag? Or do you already have data at tag 4 that you need to preserve?

Reply:
I don't, just wanted to have separate commits with tests still passing in between and so on. I can try to squash this somehow.

Reply:
Yup, no need to deal with decoding a tag 4 that never existed.